src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,5 +1,61 @@
 package frc.robot.subsystems;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import frc.robot.Constants.ArmConstants;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class IntakeSubsystem {
-    
+public class IntakeSubsystem extends SubsystemBase{
+  private CANSparkMax leftIntakeMotor, rightIntakeMotor, topIntakeMotor;
+  private double intakePower;
+  private double intakeOff = 0;
+  private double intakeOn = ArmConstants.intakeVoltage;
+  private final double HARD_STOP = 25.0;
+
+  public IntakeSubsystem() {
+    leftIntakeMotor = new CANSparkMax(ArmConstants.leftIntakeMotorID, MotorType.kBrushless);
+    rightIntakeMotor = new CANSparkMax(ArmConstants.rightIntakeMotorID, MotorType.kBrushless);
+    topIntakeMotor = new CANSparkMax(ArmConstants.topIntakeMotorID, MotorType.kBrushless);
+    leftIntakeMotor.follow(rightIntakeMotor, true);
+    leftIntakeMotor.setSmartCurrentLimit(80);
+    rightIntakeMotor.setSmartCurrentLimit(80);
+    topIntakeMotor.setSmartCurrentLimit(80);
+  }
+  public void intake () {
+    intakePower = intakeOn;
+  }
+  public void shootLow () {
+    intakePower = ArmConstants.lowShootVoltage;
+  }
+  public void shootMid () {
+    intakePower = ArmConstants.midShootVoltage;
+  }
+  public void shootHigh () {
+    intakePower = ArmConstants.highShootVoltage;
+  }
+  public void spit () {
+    intakePower = -ArmConstants.spitVoltage;
+  }
+  public void off () {
+    intakePower = intakeOff;
+  }
+  public void setIntakeMotorPower(double percent) {
+    rightIntakeMotor.set(percent); // left motor follows?
+    // leftIntakeMotor.set(percent);
+    topIntakeMotor.set(percent);
+  }
+  public double getIntakeMotorAmps() {
+    return rightIntakeMotor.getOutputCurrent();
+  }
+  private void checkIntakeAmps() {
+    if(getIntakeMotorAmps() > HARD_STOP) {
```

Comment:
Style comment: please add a newline between methods so they aren't stacked up together. For example:
```
public double getIntakeMotorAmps() {
...
}

private void checkIntakeAmps() {
...
}

```